README.md

```diff
@@ -304,6 +311,79 @@ total-size: false
 symlink-arrow: ⇒
 ```
 
+## Theme
+
+`lsd` can be configured with a theme file to set the colors.
+
+Theme can be configured in the [configuration file](#configuration)(color.theme),
+The valid theme configurations are:
+- `default`: the default color scheme shipped in `lsd`
+- theme-file-name(yaml): use the theme file to specify colors
+
+when configured with the `theme-file-name` which is a `yaml` file,
+`lsd` will look up the theme file in the following way:
+- relative name: check the XDG Base Directory
```

Comment:
Should we probably give an example filepath? Also, what is your opinion on having the themes under a `themes` subdirectory?

Also, when the user mentioned `color.theme: my-theme`, it should lookup `CONFIG_DIR/themes/my-theme.yaml`.

Reply:
actually, I was also hesitant about the `themes` subdir, maybe adding it would be a better choice considering the icon-theme plan, maybe we could also add one like `icons` for the icon configuration.

Reply:
Yeah, otherwise it will be a mess of yaml files in the root config dir if the user has a lot of themes.